app/controllers/ems_common.rb

```diff
@@ -423,17 +408,11 @@ def button
         drop_breadcrumb(:name => _("%{name} Capacity & Utilization") % {:name => @record.name},
                         :url  => show_link(@record, :refresh => "n", :display => "performance"))
         perf_gen_init_options # Intialize options, charts are generated async
-        render :update do |page|
-          page << javascript_prologue
-          page.redirect_to polymorphic_path(@record, :display => "performance")
-        end
+        javascript_redirect polymorphic_path(@record, :display => "performance")
         return
       end
       if params[:pressed] == "refresh_server_summary"
-        render :update do |page|
-          page << javascript_prologue
-          page.redirect_to  :back
-        end
+        javascript_redirect :back
```

Comment:
This one may be problematic later.. https://github.com/rails/rails/pull/22506 - Add `redirect_back` and deprecate `redirect_to :back`